### ISSUE TITLE
php71-php-fpm crash often on CentOS 7.4

### ISSUE THREAD

**Issue by tuxmaster** [remicollet/remirepo#92]
package:        php71-php-fpm-7.1.12-2.el7.remi
Where can I upload the compressed abrt crash directory, for debug it?

**Comment by remicollet** [comment]
> Where can I upload the compressed abrt crash directory, for debug it?
No idea

BTW, if you have the core file, you can retrieve the backtrace, after installing the php71-php-debuginfo package using
```
$ gdb /opt/remi/php71/root/usr/sbin/php-fpm core.xxx
(gdb) bt
```

P.S. 
- trying 7.1.13RC1 from remi-test package can also have some value.
- is this segfault new in 7.1.12 ? 
- what was the previous working version ?

**Comment by tuxmaster** [comment]
Yes, in the old version I don't have seen it.
And it will only happens with the owncloud app.
Here the back trace:

> #0  zend_get_object_type (ce=0x22) at /usr/src/debug/php-7.1.12/Zend/zend_API.c:4223
> #1  0x0000557308a64b58 in do_bind_inherited_class (op_array=<optimized out>, opline=<optimized out>, class_table=0x55730a6b34c0, parent_ce=0x7fdf7b608d00, 
>     compile_time=compile_time@entry=0 '\000') at /usr/src/debug/php-7.1.12/Zend/zend_compile.c:1156
> #2  0x0000557308ae39a5 in ZEND_DECLARE_INHERITED_CLASS_DELAYED_SPEC_VAR_HANDLER () at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:2353
> #3  0x0000557308ad1e6b in execute_ex (ex=<optimized out>) at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:429
> #4  0x0000557308a77bc4 in zend_call_function (fci=0x7fdf98416a20, fci@entry=0x7ffe2aa2d4a0, fci_cache=<optimized out>, fci_cache@entry=0x7ffe2aa2d470)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:855
> #5  0x0000557308aa66b4 in zend_call_method (object=0x7fdf98404968, obj_ce=0x7fdf98409280, fn_proxy=<optimized out>, 
>     function_name=0x7fdf98402478 "composer\\autoload\\classloader::loadclass\002", function_name_len=<optimized out>, retval_ptr=retval_ptr@entry=0x0, param_count=1, 
>     arg1=0x7fdf98416920, arg2=0x0) at /usr/src/debug/php-7.1.12/Zend/zend_interfaces.c:99
> #6  0x00005573089823de in zif_spl_autoload_call (execute_data=<optimized out>, return_value=<optimized out>) at /usr/src/debug/php-7.1.12/ext/spl/php_spl.c:420
> #7  0x0000557308a77a85 in zend_call_function (fci=fci@entry=0x7ffe2aa2d730, fci_cache=<optimized out>, fci_cache@entry=0x7ffe2aa2d700)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:869
> #8  0x0000557308a78041 in zend_lookup_class_ex (name=name@entry=0x7fdf7ca30280, key=0x7fdf80679ef0, use_autoload=use_autoload@entry=1)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:1028
> #9  0x0000557308a789d8 in zend_fetch_class_by_name (class_name=0x7fdf7ca30280, key=<optimized out>, fetch_type=fetch_type@entry=512)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:1463
> #10 0x0000557308b287c7 in ZEND_NEW_SPEC_CONST_HANDLER () at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:3199
> #11 0x0000557308ad1e6b in execute_ex (ex=<optimized out>) at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:429
> #12 0x0000557308a77bc4 in zend_call_function (fci=0x7fdf984167d0, fci@entry=0x7ffe2aa2d9a0, fci_cache=<optimized out>, fci_cache@entry=0x7ffe2aa2d970)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:855
> #13 0x00005573089bdc22 in zif_call_user_func_array (execute_data=0x7fdf98416760, return_value=0x7fdf98416730) at /usr/src/debug/php-7.1.12/ext/standard/basic_functions.c:4860
> #14 0x0000557308b2a632 in ZEND_DO_FCALL_BY_NAME_SPEC_RETVAL_USED_HANDLER () at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:876
> #15 0x0000557308ad1e6b in execute_ex (ex=<optimized out>) at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:429
> #16 0x0000557308a77bc4 in zend_call_function (fci=0x7fdf98416630, fci@entry=0x7ffe2aa2dbc0, fci_cache=<optimized out>, fci_cache@entry=0x7ffe2aa2db90)
>     at /usr/src/debug/php-7.1.12/Zend/zend_execute_API.c:855
> #17 0x00005573089bd9e2 in zif_call_user_func (execute_data=<optimized out>, return_value=0x7ffe2aa2dc50) at /usr/src/debug/php-7.1.12/ext/standard/basic_functions.c:4834
> #18 0x0000557308b2a9fe in ZEND_DO_FCALL_BY_NAME_SPEC_RETVAL_UNUSED_HANDLER () at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:797
> #19 0x0000557308ad1e6b in execute_ex (ex=<optimized out>) at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:429
> #20 0x0000557308b2d4d8 in zend_execute (op_array=op_array@entry=0x7fdf98462000, return_value=return_value@entry=0x7fdf7dd39958)
>     at /usr/src/debug/php-7.1.12/Zend/zend_vm_execute.h:474
> #21 0x0000557308a87ef3 in zend_execute_scripts (type=-1740545568, type@entry=8, retval=0x7fdf7dd39958, retval@entry=0x0, file_count=file_count@entry=3)
>     at /usr/src/debug/php-7.1.12/Zend/zend.c:1482
> #22 0x0000557308a23af0 in php_execute_script (primary_file=primary_file@entry=0x7ffe2aa302a0) at /usr/src/debug/php-7.1.12/main/main.c:2579
> #23 0x00005573088b971a in main (argc=<optimized out>, argv=<optimized out>) at /usr/src/debug/php-7.1.12/sapi/fpm/fpm/fpm_main.c:1966

**Comment by remicollet** [comment]
> Yes, in the old version I don't have seen it.

But what was the previous version ?

**Comment by tuxmaster** [comment]
Sorry, my mistake.
7.1.11 was the last working one.

**Comment by remicollet** [comment]
Can this issue be reproduced easily ?
Have you a minimal reproducer ?
Please give a try to 5.1.13RC1 (in remi-test) to ensure it is not already fixed, else please file an [upstream bug](https://bugs.php.net/) so this regression can be properly tracked

**Comment by tuxmaster** [comment]
No, I can't reproduce it easily. I can only say, that it will happens when using the owncloud website.
Do you mean 7.1.13RC1?

**Comment by remicollet** [comment]
> Do you mean 7.1.13RC1?
Yes

**Comment by tuxmaster** [comment]
It was fixed by switching to php 7.2
